common/test/acceptance/pages/lms/teams.py

```diff
@@ -324,6 +324,64 @@ def cancel_team(self):
         self.q(css='.create-team .action-cancel').first.click()
         self.wait_for_ajax()
 
+    def click_membership_button(self):
+        """Clicks the 'edit membership' button"""
+        self.q(css='.action-edit-members').first.click()
+        self.wait_for_ajax()
+
+    @property
+    def membership_button_present(self):
+        return self.q(css='.action-edit-members').present
+
+
+class EditMembershipPage(CoursePage):
+    """
+    Staff or discussion-privileged user page to remove troublesome or inactive
+    students from a team
+    """
+    def __init__(self, browser, course_id, team):
+        """
+        Set up `self.url_path` on instantiation, since it dynamically
+        reflects the current team.
+        """
+        super(EditMembershipPage, self).__init__(browser, course_id)
+        self.team = team
+        self.url_path = "teams/#teams/{topic_id}/{team_id}".format(
```

Comment:
The url_path is actually more complicated-- it ends with edit-team/manage-members. So I'm not sure how this is working.